ext/ffi/00_ffi.js

```diff
@@ -326,6 +330,33 @@
             },
           );
         }
+
+        if (needsUnpacking && !isNonBlocking) {
+          const call = this.symbols[symbol];
+          const parameters = symbols[symbol].parameters;
+          const v = new Int32Array(2);
+          const b = new BigInt64Array(v.buffer);
+          if (parameters.length == 0) {
+            this.symbols[symbol] = function () {
+              call(v);
+              return b[0];
+            };
+          } else {
+            // Make sure V8 has no excuse to not optimize this function.
+            eval(
+              `this.symbols[symbol] = function (${
+                parameters.map(
+                  (_, index) => `p${index}`,
+                ).join(", ")
+              }) {
+                call(${
+                parameters.map((_, index) => `p${index}`).join(", ")
+              }, v);
+                return b[0];
+              }`,
+            );
```

Comment:
There be dragons 🐉

Reply:
imaginary dragons 🤷 

its just generates argument param names based on `index` :) unfortunately spread params don't let `call` optimize